### ISSUE TITLE
SpringBoard was not recognised

### ISSUE THREAD

**Issue by Kikobeats** [omrilotan/isbot#231]
User Agent String
-------

```
SpringBoard/50 CFNetwork/1209 Darwin/20.2.0
SpringBoard/50 CFNetwork/1240.0.4 Darwin/20.6.0
SpringBoard/50 CFNetwork/1220.1 Darwin/20.3.0
SpringBoard/50 CFNetwork/1206 Darwin/20.1.0
SpringBoard/50 CFNetwork/1197 Darwin/20.0.0
```

Reproduce
-------

to be honest I'm not sure what is it. It seems it's the user agent used by Apple devices:
https://github.com/search?q=SpringBoard%2F50&type=code

maybe it's a legit browser agent 🤔

**Comment by omrilotan** [comment]
Please see https://github.com/omrilotan/isbot/pull/233

**Comment by omrilotan** [comment]
Resolved in [4.1.1](https://github.com/omrilotan/isbot/blob/next/CHANGELOG.md#411).
Updating from version 3? Please see [migration instructions](https://github.com/omrilotan/isbot/blob/next/CHANGELOG.md#411)
